client_linux_test.go

```diff
@@ -316,38 +316,3 @@ func TestUnpackServiceCrashers(t *testing.T) {
 		_ = unpackService(&svc)([]byte(crash))
 	}
 }
-
-func TestServiceRoundTrip(t *testing.T) {
```

Comment:
What's the reason for removing this test?

And what do you think about adding the test from #1?

Reply:
It no longer parses because the size of the type changed.

I might add the test from #1 in a larger change that adds a separate suite that actually calls into a virtualized kernel.